modules/auth/auth.service.js

```diff
@@ -17,12 +18,18 @@ const login = async (body) => {
   const existedUser = await userService.findByUsername(body.username);
 
   if (!existedUser) {
-    throw new Error('Username is not existed!');
+    throw new APIError({
+      message: 'Username is not existed!',
+      status: 400
+    });
   }
 
   const comparePasswordResult = await bcrypt.compareSync(body.password, existedUser.password);
   if (!comparePasswordResult) {
-    throw new Error('Password is not correct!');
+    throw new APIError({
+      message: 'Password is not correct!',
+      status: 400
```

Comment:
as above

Reply:
Password is not correct is 400 because this error come from user input, we cannot say with client that your password does not exist, right!

Reply:
okay